pages/price-feeds/use-real-time-data/aptos.mdx

```diff
@@ -1,31 +1,87 @@
 ---
-description: Consume Pyth Network prices in applications on Aptos
+description: Consume Pyth Network prices in Aptos applications
 ---
 
-# Pyth on Aptos chains
+import { Callout } from "nextra/components";
 
-Pyth is available on Aptos Mainnet, Aptos Testnet, and Movement devnet.
+# How to Use Real-Time Data in Aptos Contracts
 
-Aptos contracts can update and fetch the Pyth prices using the Pyth Aptos Contract, which has been deployed on Mainnet. The documented source code can be found [here](https://github.com/pyth-network/pyth-crosschain/blob/main/target_chains/aptos/contracts/sources/pyth.move).
+This guide explains how to use real-time Pyth data in Aptos applications.
 
-## Updating Price Feeds
+## Configuring the `Move.toml` file
 
-The mechanism by which price feeds are updated on Aptos is explained [here](./pythnet-price-feeds.md). The [pyth-aptos-js](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/aptos/sdk/js) package can be used to fetch price feed update data which can be passed to the `pyth::update_price` on-chain function.
+Add the Pyth Contract to your project dependencies in the `Move.toml` file:
 
-## Example Applications
+```toml copy
+[dependencies]
+Pyth = { git = "https://github.com/pyth-network/pyth-crosschain.git", subdir = "target_chains/aptos/contracts", rev = "main" }
+```
 
-- [Minimal on-chain contract](https://github.com/pyth-network/pyth-crosschain/blob/main/target_chains/aptos/examples/fetch_btc_price/sources/example.move) which updates and returns the Pyth BTC/USD price.
-- [Full-stack React app and on-chain contract](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/aptos/examples/mint_nft) which uses the [pyth-aptos-js](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/aptos/sdk/js) package to update the price used by the contract.
-- [In-depth explanation](https://youtu.be/0b0RXi41pN0) of how Pyth works on Aptos and how to integrate Pyth data in your application from one of our contributor.
+The named addresses of `pyth`, `wormhole`, and `deployers` must be defined at compile time. These addresses are used to interact with the Pyth contract on Aptos.
 
-## Contract Addresses
+```toml copy
```

Comment:
maybe leave a comment like `# Aptos mainnet addresses, see [link] for other options` ?

Reply:
It's already mentioned after the code block.